### DISCUSSION TITLE
Dev productivity - Quarkus CLI

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#34303] [Quarkus Blog/Website]
# Dev productivity - Quarkus CLI

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-cli/

**Comment by bulldog98** [comment]
This is a really nice article.

I think that `quarkus plug list --installable` returns quarkus (cli) itself should be considered a bug, to me it makes no sense whatsoever to try to install it as a plugin to itself
